packages/clarity-js/src/interaction/scroll.ts

```diff
@@ -71,6 +72,10 @@ function recompute(event: UIEvent = null): void {
     timeout = setTimeout(process, Setting.LookAhead, Event.Scroll);
 }
 
+const throttledRecompute = throttle((event: UIEvent = null) => {
+    recompute(event);
```

Comment:
How does this get executed? What if multiple events are called before the delay time?